### DISCUSSION TITLE
cpu/reference

### DISCUSSION THREAD

**Discussion by giscus** [jielahou/blog-comments#1] [Announcements]
# cpu/reference

备战NSCSCC的过程中，势必要阅读很多资料。在此做整理，以减少学弟学妹搜集资料的时间。个人是负责SOC和部分系统移植的，所以主要是这两个方向的资料。 UCAS NSCSCC (ucas-nscscc.github.io)：中科大NSCSCC教程，初步讲解了Block Design、上板等过程，适合作为入门第一步资料 重庆大学操作系统实验文档 (cqu.ai)：如何编写裸机程序 基于龙芯FPGA开发板的计算机系统综合实验(https://oscourse-tsinghua.github.io/LoongsonCsprj2020/cpu/) 串口烧写SPI Flashhttps://chiplab.readthedocs.io/zh/latest/FPGA_run_linux/flash.html 给定十六进制字符串(e.g.0x3C1D9FC5)，转换成MIPS指令eg.bucknell.edu/~csci320/mips_web/

http://jielahou.com/cpu/reference.html

**Comment by SeddonShen** [comment]
支持博主，前排学习，不过请注意UCAS是国科大不是中科大哦～

**Reply by jielahou** [reply]
感谢指出，已经fix : )
话说我看您好眼熟啊，印象中14年那会玩建站，老哥是不是还运营了一个联盟来着: )，一眨眼十年过去了，还是挺怀念的。
